wgimson/QRdly
# pwa native mobile functionality added (how to test in comments)

@alexgimson86  - The app now functions as a native app on mobile, has a service worker for caching, etc.
``
**-- TO TEST --**

1.   pull down this branch 
2.  run` npm start`
3.  In Chrome right click and open inspector. In the far left corner select mobile view
4. ![image](https://user-images.githubusercontent.com/676537/119440958-3fe55100-bcf3-11eb-91e8-b5f029e9249a.png)
5.Then Select the Lighthouse tab on the far right.
6. ![image](https://user-images.githubusercontent.com/676537/119441021-660af100-bcf3-11eb-81cb-655d889c205a.png)
7. the PWA report should pass for everything except using HTTPS, which will be implemented when we deploy.
